mappings/net/minecraft/world/poi/PointOfInterestStorage.mapping

```diff
@@ -10,15 +11,15 @@ CLASS net/minecraft/class_4153 net/minecraft/world/poi/PointOfInterestStorage
 		ARG 2 predicate
 	METHOD method_19118 getDistanceFromNearestOccupied (Lnet/minecraft/class_4076;)I
 		ARG 1 pos
-	METHOD method_19119 get (Ljava/util/function/Predicate;JLnet/minecraft/class_4153$class_4155;)Ljava/util/stream/Stream;
+	METHOD method_19119 getInChunkSection (Ljava/util/function/Predicate;JLnet/minecraft/class_4153$class_4155;)Ljava/util/stream/Stream;
```

Comment:
Why not `getInSubchunk` (you said it's called this in the tech community)

Reply:
I think renaming section to subchunk should be done in another PR. But I disagree with that, since "section" is present in NBT tags, debug screen, exception/log messages, Minecraft wiki.

Reply:
I go by the standard currently set by the yarn that I'm merging into, not by some discussion in an issue that was opened 5 minutes ago.